maui/eda/eda.py

```
@@ -470,18 +474,35 @@ def daily_distribution_analysis(
     >>> fig = eda.daily_distribution_analysis(df, 'dt', 'landscape')
     """
 
+    # Convert with errors='coerce' to transform invalid dates to NaT (Not a Time)
+    df_original_size = len(df)
+    df[date_column] = pd.to_datetime(df[date_column], errors='coerce')
+
+    # Check for invalid dates and generate warning
+    invalid_dates_count = df[date_column].isna().sum()
+    if invalid_dates_count > 0:
+        warnings.warn(
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)